packages/studio/frontend/src/views/FlowBuilder/nodeProps/ConditionModalForm.tsx

```diff
@@ -1,11 +1,11 @@
 import { FlowView } from '@botpress/common'
-import { SuperInput, SiTypes } from '@botpress/inspector-kit'
 import _ from 'lodash'
 import React, { Component } from 'react'
 import { Button, Radio, FormControl, Alert, Form } from 'react-bootstrap'
 import { connect } from 'react-redux'
 import Select from 'react-select'
 
+import { SuperInput, SiTypes } from '~/src/components/SuperInput'
```

Comment:
We don't use aliases anymore. So you need to use relative paths.

Reply:
I don't like having to guess how many sub-folders i'm in to write an import. Can we just re-add aliases? I saw this on parcel's documentation so I think it's ok to use this whenever importing something at the src root.
https://en.parceljs.org/module_resolution.html#absolute-paths

Reply:
If you can get them to work and see a benefit, I don't see a downside. So you can re-add them, yes!